docs/stack.md

```diff
@@ -1,42 +1,64 @@
-## Stack
+# Stack
 
-### HTML
+## HTML
 
 HTML for the majority of the site is generated from Django/Wagtail templates and components.
 
-### Frontend styleguide
+## Frontend styleguide
 
 This site has a [styleguide](https://foundation.mozilla.org/en/style-guide/), which can be refrenced for frontend development to note intended usage of exisitng styles and components.
 
-### CSS
+## CSS
 
-CSS is generated from [Sass](http://sass-lang.com/). Styles are based on the mofo-bootstrap theme found under `source/sass/mofo-bootstrap`.
+CSS is currently very messy.
```

Comment:
We do have an epic about updating our usage of CSS for the foundation site and I think linking the epic would be a better representation of the ongoing situation. 

In my opinion, "messy" to me means we don't have a future or current todo list or course of action and I do not think that is the case vs not having time/people.

Reply:
I have changed the sentence and added a link to the epic at the end of the paragraph in e1f41ccfc4ee5a1b73c22000698e69b138e444e4. 

The epic does feel outdated though. The title only speaks of "Transition away from Bootstrap" while the description says "Decide whether we want to move to Tailwind. If so, migrate and get rid of SASS". I guess we could potentially update the title and description to better reflect the current state of the plan.

Reply:
Yeah I think updating the epics title/description would probably solve this problem since I do agree that is the outdated part. thanks for addressing this part